weapons-of-mass-deduction/chess-app
# Push update to client

What I did:
1. Installed the Pusher gem to push messages to user.
2. Updated gameplay to only allow players to move pieces of their own color.
3. When a player makes a move, opponent's browser will refresh automatically. 

To test:
Open 2 windows, log in as 2 different users and play on the same game.

Good to have:
Instead of refreshing the page, the opponent's browser should make a ajax request to the backend to retrieve the latest game data.
